src/Common/Storage/Commands.Storage/File/Cmdlet/GetAzureStorageFileContent.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet
     using LocalDirectory = System.IO.Directory;
     using LocalPath = System.IO.Path;
 
-    [Cmdlet(VerbsCommon.Get, LocalConstants.FileContentCmdletName, SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = LocalConstants.ShareNameParameterSetName)]
```

Comment:
GetAzureStorageFileContent may need to overwrite local file, should keep SupportsShouldProcess.